src/main/resources/avro/bdg.avdl

```diff
@@ -57,8 +57,55 @@ record Contig {
   union { null, string } species = null;
 }
 
+record Sequence {
+  union { null, string } bases;
+  union { null, string } qualities;
+}
+
+/**
+   The DNA fragment that is was targeted by the sequencer, resulting in
+   one or more reads.
```

Comment:
I think this description could be improved to show the generality of this object.  Perhaps add a note about how this can refer to a "bucket" of reads that derive from the same originating molecule, or that it can model reads generated from a single library member or cluster (in Illumina terminology).